### PR TITLE
Add Booking link on ShipmentTransport

### PR DESCRIPTION
**Pull request by HenrikHL**

### CODE REVIEW

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 1]
```diff
@@ -638,6 +638,7 @@ CREATE TABLE dcsa_im_v3_0.transport_plan_stage_type (
 
 DROP TABLE IF EXISTS dcsa_im_v3_0.shipment_transport CASCADE;
 CREATE TABLE dcsa_im_v3_0.shipment_transport (
+    booking_id uuid NOT NULL REFERENCES dcsa_im_v3_0.booking(id),
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:641]:
```suggestion
    booking_id uuid NULL REFERENCES dcsa_im_v3_0.booking(id),
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 2]
```diff
@@ -638,6 +638,7 @@ CREATE TABLE dcsa_im_v3_0.transport_plan_stage_type (
...
     shipment_id uuid NOT NULL REFERENCES dcsa_im_v3_0.shipment(id),
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql:642]:
```suggestion
    shipment_id uuid NULL REFERENCES dcsa_im_v3_0.shipment(id),
```

File: datamodel/initdb.d/03_dcsa_im_v3_0.sql [hunk 3]
```diff
@@ -638,6 +638,7 @@ CREATE TABLE dcsa_im_v3_0.transport_plan_stage_type (
 
 DROP TABLE IF EXISTS dcsa_im_v3_0.shipment_transport CASCADE;
 CREATE TABLE dcsa_im_v3_0.shipment_transport (
+    booking_id uuid NULL REFERENCES dcsa_im_v3_0.booking(id),
     shipment_id uuid NOT NULL REFERENCES dcsa_im_v3_0.shipment(id),
```
Comment (HenrikHL) [inline datamodel/initdb.d/03_dcsa_im_v3_0.sql]:
```suggestion
    booking_id uuid NULL REFERENCES dcsa_im_v3_0.booking(id),
    shipment_id uuid NULL REFERENCES dcsa_im_v3_0.shipment(id),
```

**Comment by HenrikHL** [comment]
For info - ShipmentTransport now has a bookingID link. It is necessary to add Mode_of_transport for preCarriage already at booking level...

**Review by mp-assco — APPROVED** [review]

**Review by pimg — APPROVED** [review]

**Pull request merged**